msg/MpcStatistics.msg

```diff
@@ -0,0 +1,11 @@
+# This message provides essential diagnostics information
+
+time stamp
+
+uint16 num_iterations
+float32 problem_update_time
+float32 solve_time
+float32 final_cost
+float32 regularization_rate
```

Comment:
```suggestions
uint16 iterations
float32 total_time
float32 solve_time
float32 cost
float32 regularization
```

It is not clear to me what it is the difference between problem_update_time and solve_time.
Also, I think it is confusing terms final and rate.
Final might suggest we are solving the OC problem (actually it is more like a current cost in this sense)
Instead, rate suggest that it is a factor use to decrease or increase the regularization.

Reply:
I initially meant problem update time is the time taken to update all references, constraints, etc. - everything that is done _before_ calling solve. I've now changed it to total time though.